### PR TITLE
Update doc line for length according to style guide

### LINKED ISSUE #447: Fix style issue `Doc line too long FLK-W505`
Fix `Doc line too long  FLK-W505`: https://deepsource.io/gh/scanapi/scanapi/issue/FLK-W505/occurrences

image

> Docstring line lengths are recommended to be no greater than 79 characters.

Child of https://github.com/scanapi/scanapi/issues/405

### PR DESCRIPTION
**Pull request by djs-djs**
## Description
Reduced doc line length to under 79 characters to fit styling conventions 

## Motivation behind this PR?
FLK-W505: 
https://github.com/scanapi/scanapi/issues/447
https://deepsource.io/gh/scanapi/scanapi/issue/FLK-W505/occurrences

## What type of change is this?
Bug Fix 

## Checklist
 

- [ ]  I have added a changelog entry / my PR does not need a new changelog entry. [Instructions](https://github.com/scanapi/scanapi/wiki/Changelog).
- [ ] I have added/updated unit tests. [Instructions](https://github.com/scanapi/scanapi/wiki/Writing-Tests).
- [ ] New and existing unit tests pass locally with my changes. [Instructions](https://github.com/scanapi/scanapi/wiki/Run-ScanAPI-Locally#tests)
- [ ] I have self-documented code my changes by adding docstring(s) and comment(s). [Instructions](https://github.com/scanapi/scanapi/wiki/First-Pull-Request#7-make-your-changes)
- [ ] Current PR does not significantly decrease the code coverage and docstring coverage.
- [x] My code follows the style guidelines of this project.
- [ ] I have run ScanAPI locally and manually tested my changes. [Instructions](https://github.com/scanapi/scanapi/wiki/Run-ScanAPI-Locally).

## Issue
Closes #447

### CODE REVIEW

File: scanapi/errors.py [hunk 1]
```diff
@@ -40,7 +40,8 @@ def __init__(self, error_message, code, *args):
 
 
 class BadConfigurationError(Exception):
-    """Raised when an environment variable was not set or was badly configured"""
+    """Raised when an environment variable was not set
```
Comment (camilamaia) [inline scanapi/errors.py:43]:
```suggestion
    """Raised when an environment variable was not set
```

File: scanapi/errors.py [hunk 2]
```diff
@@ -40,7 +40,8 @@ def __init__(self, error_message, code, *args):
 
 
 class BadConfigurationError(Exception):
-    """Raised when an environment variable was not set or was badly configured"""
+    """Raised when an environment variable was not set 
+    or was badly configured"""
```
Comment (camilamaia) [inline scanapi/errors.py:44]:
```suggestion
    or was badly configured
    """
```

**Review by camilamaia — COMMENTED** [review]
Hey, thanks for the contribution! I left some suggestions to make the docstring pass in the checks.

But DeepSource is also complaining about the missing docstring for the MalformedSpecError class. Would you mind adding it?

https://deepsource.io/gh/scanapi/scanapi/run/523e6b46-f6ea-4ea3-a2c6-6611fae692dc/python/PY-D0002

Here you can find how we create the docstrings: https://github.com/scanapi/scanapi/wiki/First-Pull-Request#7-make-your-changes

Thanks again!

**Comment by djs-djs** [comment]
@camilamaia Apologies for all the pushs; I am a noob, lol. 

Edited docstring of BadConfigurationError per comments. Also added docstring to MalformedSpecError. Comments/suggestions for edits are welcome and encouraged :) 

Thanks again for the opportunity!

File: scanapi/errors.py [hunk 3]
```diff
@@ -1,4 +1,7 @@
 class MalformedSpecError(Exception):
+    """Raised when API spec is invalid;
+    base class for other exceptions
+    """
```
Comment (camilamaia) [inline scanapi/errors.py:4]:
```suggestion
    """
    
```

**Review by camilamaia — COMMENTED** [review]
Just one minor suggestions and it should be good to go 🚀 

Thank you!

**Comment by djs-djs** [comment]
Thank you! Will do

**Comment by camilamaia** [comment]
DeepSource check is still failing. If you click on `details` you can see what needs to be changed.

image

And then click on `here`:

image

image

The first one `Unnecessary pass statement` you can ignore for now. The `1 blank line required after class docstring` is the one you need to fix.

I hope it is clear, sorry for the confusion

**Comment by djs-djs** [comment]
Thanks for being patient with me! The only issue now is the Unnecessary pass statement which you said to ignore for now. Please let me know if you need anything else :)

**Review by camilamaia — APPROVED** [review]
No worries, we are here together! Thank YOU for the effort! LGTM now ✅

**Pull request merged**